common/button/button-en.html

```diff
@@ -0,0 +1,426 @@
+<button type="button" class="btn btn-default">8</button>
+</div>
+</div>
+
+<h4>Code sample</h4>
+<pre><code>// Button toolbar:
+&lt;div class="<strong>btn-toolbar</strong>" role="toolbar" aria-label="Toolbar with button groups"&gt;
+	&lt;div class="btn-group" role="group" aria-label="First group"&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		1
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		2
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		3
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		4
+		&lt;/button&gt;
+	&lt;/div&gt;
+
+	&lt;div class="btn-group" role="group" aria-label="Second group"&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		5
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		6
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		7
+		&lt;/button&gt;
+	&lt;/div&gt;
+
+	&lt;div class="btn-group" role="group" aria-label="Third group"&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		8
+		&lt;/button&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3 id="inputgroups">Input Groups</h3>
+<p>Easily extend form controls by adding buttons, or button groups on either side of textual inputs with the <code>input-group</code> and <code>input-group-btn</code> classes.</p>
+<h4>Working Example</h4>
+<div class="input-group mb-3">
+<div class="input-group-btn">
+<button class="btn btn-default" type="button">Button</button>
+</div>
+<input type="text" class="form-control">
+</div>
+
+<div class="input-group mb-3">
+<input type="text" class="form-control">
+<div class="input-group-btn">
+<button class="btn btn-default" type="button">Button</button>
+</div>
+</div>
+
+<h4>Code Sample</h4>
+<pre><code>// Input groups:
+&lt;div class="<strong>input-group</strong> mb-3"&gt;
+	&lt;div class="<strong>input-group-btn</strong>"&gt;
+		&lt;button class="btn btn-default" type="button"&gt;Button&lt;/button&gt;
+	&lt;/div&gt;
+	&lt;input type="text" class="form-control"&gt;
+&lt;/div&gt;
+
+&lt;div class="<strong>input-group</strong> mb-3"&gt;
+	&lt;input type="text" class="form-control"&gt;
+	&lt;div class="<strong>input-group-btn</strong>"&gt;
+		&lt;button class="btn btn-default" type="button"&gt;Button&lt;/button&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3 id="datatoggle">Data-toggle buttons</h3>
+
+<div class="alert alert-info">
+	<p>Currently, this feature is not functional as we do not implement Bootstrap's JavaScript.</p>
+</div>
+
+<p>Bootstrap’s <code>.button</code> styles can be applied to other elements, such as <code>label</code>s, to provide checkbox or radio style button toggling. Add <code>data-toggle="buttons"</code> to a <code>.btn-group</code> containing those modified buttons to enable their toggling behavior via JavaScript and add <code>.btn-group-toggle</code> to style the <code>input</code>s within your buttons.</p>
+
```

Comment:
Add a link to the bootstrap working example - https://getbootstrap.com/docs/3.4/components/#dropdowns

And add a mention about this wet-boew PR which are  going to add a similar functionality in the future - https://github.com/wet-boew/wet-boew/pull/9532

Reply:
Changes made in the commit "Addressed Requested Changes".